pkgs/applications/social/patchwork/default.nix

```diff
@@ -0,0 +1,38 @@
+{ stdenv, appimage-run, fetchurl, runtimeShell, gsettings-desktop-schemas, gtk3, gobject-introspection, wrapGAppsHook }:
+
+let
+  version = "3.11.6";
+in
+
+stdenv.mkDerivation rec {
+  name = "patchwork-${version}";
+
+  src = fetchurl {
+    url = "https://github.com/ssbc/patchwork/releases/download/v${version}/Patchwork-${version}-linux-x86_64.AppImage";
+    sha256 = "1d2rq6l2mbi6bqv8321ps5bi9n8aqriwv7agc2zvgylm0cxyc4cq";
+  };
+
+  nativeBuildInputs = [ wrapGAppsHook ];
+  buildInputs = [ appimage-run gtk3 gsettings-desktop-schemas gobject-introspection ];
```

Comment:
From your log I think you are missing `git-ssb` from `nodePackages`.

Reply:
It looks like an error, but `git-ssb` is optional and the error is only being output for debugging. I'll prioritize fixing that debug output so that it's more obvious.